x/smart/psiphon_dialer.go

```diff
@@ -0,0 +1,23 @@
+// +build psiphon
+// If the build tag `psiphon` is set, allow importing and calling psiphon
+
+package smart
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/Jigsaw-Code/outline-sdk/transport"
+	"github.com/Jigsaw-Code/outline-sdk/x/psiphon"
+)
+
+func getPsiphonDialer(ctx context.Context, psiphonJSON []byte) (transport.StreamDialer, error) {
```

Comment:
Can you name it with new instead of get, since we are creating it?

Reply:
done